### PR TITLE
Minor changes and Sidecar page update

### PR DESCRIPTION
**Pull request by hyd628**
### Description/Original PRs

https://github.com/PureStake/moonbeam-docs/pull/253
https://github.com/PureStake/moonbeam-docs/pull/259

### Checklist

- [x] If this requires removing old images from the `moonbeam-docs` repo, I have created a corresponding PR
- [x] If this requires adding/updating/deleting redirects, I have created a corresponding PR in the `moonbeam-mkdocs` repo
- [x] If this requires removing old variables from the `moonbeam-docs` repo, I have created a corresponding PR

### Corresponding PRs

Please link to any additional corresponding PRs here.

### CODE REVIEW

File: builders/tools/sidecar.md [hunk 1]
```diff
@@ -54,12 +54,12 @@ node_modules/.bin/substrate-api-sidecar --version
     export SAS_SUBSTRATE_WS_URL=wss://wss.api.moonbase.moonbeam.network
     ```
 
-=== "Moonbeam开发节点"
+=== "Moonbeam Dev Node"
```
Comment (eshaben) [inline builders/tools/sidecar.md:57]: this should remain as it was, it seems like we have it as `=== "Moonbeam开发节点"` everywhere else

File: node-operators/networks/tracing-node.md [hunk 2]
```diff
@@ -74,7 +74,8 @@ Geth的`debug`和`txpool` API以及OpenEthereum的`trace`模块提供一个非
   - **`--ethapi=debug`** — 选择性标识，启用`debug_traceTransaction`、`debug_traceBlockByNumber`和`debug_traceBlockByHash`
   - **`--ethapi=trace`** — 选择性标识，启用`trace_filter` 
   - **`--ethapi=txpool`** — 选择性标识，启用`txpool_content`、`txpool_inspect`和`txpool_status`
-  - **`--wasm-runtime-overrides=/moonbeam/<network>-substitutes-tracing`** — 用于追踪指定存储本地WASM runtime路径的**必备**标识。接受网络作为参数`moonbeam`、`moonriver`或`moonbase``moonbase`（用于开发节点和 Moonbase Alpha）
+  - **`--wasm-runtime-overrides=/moonbeam/<network>-substitutes-tracing`** — **必备** 用于追踪指定存储本地WASM runtime路径的标识。接受网络作为参数`moonbeam`、`moonriver`或`moonbase``moonbase`（用于开发节点和 Moonbase Alpha）
```
Comment (eshaben) [inline node-operators/networks/tracing-node.md:77]: looks like there is an extra `moonbase` here but not sure

**Review by eshaben — CHANGES_REQUESTED** [review]

**Comment by hyd628** [comment]
Thanks @eshaben . Fixed issues.

**Review by eshaben — APPROVED** [review]
Thanks Henry! LGTM ✨

**Pull request merged**